README.md

```diff
@@ -53,11 +53,11 @@ Your views should be node modules that export a React component. Let's assume yo
 ```js
 var React = require('react');
 
-var HelloMessage = React.createClass({
-  render: function() {
+class HelloMessage extends React.Component {
+  render() {
     return <div>Hello {this.props.name}</div>;
   }
-});
+};
```

Comment:
Sorry, missed this before - can you also remove the semicolons at the end of each of the classes?